test/builders/build/runtime-upgrades.js

```diff
@@ -14,21 +14,21 @@ describe('Runtime Upgrades', () => {
       const api = await getApi('wss://wss.api.moonbase.moonbeam.network');
       const runtime = await api.query.system.lastRuntimeUpgrade();
       // Assert the runtime is equal to the latest version we have on the docs
-      assert.equal(runtime.toJSON().specVersion, 3000);
+      assert.equal(runtime.toJSON().specVersion, 3100);
       api.disconnect();
     });
     it('should return the latest runtime version for Moonriver', async () => {
       const api = await getApi('wss://wss.api.moonriver.moonbeam.network');
       const runtime = await api.query.system.lastRuntimeUpgrade();
       // Assert the runtime is equal to the latest version we have on the docs
-      assert.equal(runtime.toJSON().specVersion, 3000);
+      assert.equal(runtime.toJSON().specVersion, 3100);
       api.disconnect();
     });
     it('should return the latest runtime version for Moonbeam', async () => {
       const api = await getApi('wss://wss.api.moonbeam.network');
       const runtime = await api.query.system.lastRuntimeUpgrade();
       // Assert the runtime is equal to the latest version we have on the docs
-      assert.equal(runtime.toJSON().specVersion, 3001);
+      assert.equal(runtime.toJSON().specVersion, 3100);
       api.disconnect();
```

Comment:
Moonriver and Moonbeam haven't been upgraded to 3100 yet. So Moonriver is still on 3000 and Moonbeam is still on 3001. You can check this on Subscan. The release schedule is in the RT3100 ticket on Jira for reference